GVRf/Framework/src/org/gearvrf/GVRRenderTexture.java

```diff
@@ -52,6 +52,31 @@ public GVRRenderTexture(GVRContext gvrContext, int width, int height,
         super(gvrContext, NativeRenderTexture.ctorMSAA(width, height,
                 sampleCount));
 
+    /**
```

Comment:
Not really. We are trying to add javadoc as much as possible. Patch formality is flexible in this project.